docs/developers/bots.md

```diff
@@ -1,8 +1,54 @@
+
+```shell title="example.env"
+PRIVATE_KEY=0xf7cbade2b9eec8fc83aa70e4b43f480d0ca78b7060737ead2669d095f2035322
+COMMON_PRICE_FEED_CONFIG='"commonPriceFeedConfig": {"cryptowatchApiKey": "abcd","tradermadeApiKey": "efg","quandlApiKey": "hijklmnop"}'
+COMMAND=yarn optimistic-oracle --network mainnet_privatekey
+```
+
+Once you have a properly configured `.env` file, use the following commands to pull the Docker image and run a container with your specified configuration.
+
+```shell
+# Pull the latest docker container image
+docker pull umaprotocol/protocol:latest
+
+# Start the optimistic oracle bot Docker container
+docker run --name optimistic-oracle-bot -d --env-file ./example.env umaprotocol/protocol:latest
+# *your container hash should print here*
+
+# List logs from running bot:
+docker logs <your container hash>
+```
+
+Your `optimistic-oracle` bot should now be running, please read on to find out about how it works and how to [configure](developers/bot-param.md) it.
+
+## Motivation
+
+The bots described below are infrastructure tools that help maintain the overall health of the UMA ecosystem. They can be very useful for project creators, synthetic token holders, and third-party keepers.
+* The `optimistic-oracle` bot ([implementation](https://github.com/UMAprotocol/protocol/tree/master/packages/optimistic-oracle)) makes it easy to provide quick and accurate responses to price requests to the Optimistic Oracle, and to dispute incorrect price proposals.
+* The `liquidator` bot ([implementation](https://github.com/UMAprotocol/protocol/tree/master/packages/liquidator)) liquidates undercollateralized positions in `ExpiringMultiParty` contracts.
+* The `disputer` bot ([implementation](https://github.com/UMAprotocol/protocol/tree/master/packages/disputer)) disputes incorrect liquidations of `ExpiringMultiParty` positions.
+* The `monitors` bot ([implementation](https://github.com/UMAprotocol/protocol/tree/master/packages/monitors)) can monitor synthetic prices, collateralization ratios, wallet balances, and other details that may be important to a user.
+
+## Incentives to Running an Optimistic Oracle Bot
+
+Contracts making a price request to the Optimistic Oracle may offer a `proposerReward` paid out to a price proposer, usually a keeper running an `optimistic-oracle` bot. Project creators may also expect to propose their own prices, in which case there may be no proposer reward, but running an `optimistic-oracle` bot can make their lives easier.
+
+The `optimistic-oracle` bot will also automatically dispute any prices that appear to be different from the true price as calculated by the price feed, outside of a user-specified allowable margin of error. A dispute will trigger a vote on the actual price in the DVM. If UMA tokenholders vote in favor of the disputer, the disputer will earn the proposer's bond (equal to the final fee for the collateral type used by the contract, ~$400) plus any additional bond required by the price requestor.
```

Comment:
nit: the default value is the final fee, but the price requestor could set it lower technically.

Might be more accurate to just say they get the proposer's bond which defaults to the final fee, but is often more.

Reply:
will make that update